toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -483,6 +481,47 @@ chroot_and_install_rpms gperf
 chroot_and_install_rpms cryptsetup
 build_rpm_in_chroot_no_install systemd
 
+build_rpm_in_chroot_no_install golang-1.13
+build_rpm_in_chroot_no_install groff
```

Comment:
Are all these packages new additions to the toolchain?

Reply:
Yes.  They were needed as dependencies in order to add libsemanage to the toolchain, because shadow-utils picked up a dependency on libsemanage.  I asked @anphel31 about it, and he said that adding in the needed dependencies for libsemanage was the way to go.

Reply:
Does this mean that these packages should be added to the toolchain manifest as well?

Reply:
I don't know the answer to that question, as I don't fully understand the context of what needs to go in the toolchain manifest.  @anphel31 can you help here?  If I added packages into build_official_toolchain_rpms.sh solely in order to provide dependencies for other toolchain packages (not because they themselves are needed as part of the toolchain), do I also need to add them to the toolchain manifests?

Reply:
I believe Mateusz is right - we need to mention every package (and its subpackages) we build in the toolchain inside these manifests. It's tedious and annoying - we know. Joe is actually providing automation for that part, so the manifests should go away soon. But probably not before this change gets in.:)

---
In reply to: [498809836](https://github.com/microsoft/CBL-Mariner/pull/100#discussion_r498809836) [](ancestors = 498809836)

Reply:
Okay, I'll make the change to add those packages and remove util-linux as mentioned elsewhere.  Once I do that I should probably rerun the toolchain validation pipelines to make sure I didn't make a mistake, so that will take at least 9 hours assuming it's successful on the first run.

Reply:
Actually, I don't think util-linux needs removal after all.  I got an e-mail about a comment to that effect, but I can't find it here.  Regardless, it looks to me like util-linux is still in the toolchain, and therefore should remain in the manifests.  I believe that packages in the manifests that weren't built would have been caught by the toolchain pipelines.  It seems as though they don't validate that all built packages are in the manifests though.

Reply:
Fixed.  I've kicked off buddy pipeline builds for both toolchains to verify that I didn't make any mistakes in adding them:

https://dev.azure.com/mariner-org/mariner/_build/results?buildId=36188&view=results
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=36189&view=results